src/Raven.Server/Web/Authentication/AdminCertificatesHandler.cs

```diff
@@ -93,6 +93,14 @@ public async Task Generate()
                     throw new InvalidOperationException($"Cannot generate the client certificate '{certificate.Name}' with 'Cluster Admin' security clearance because the current client certificate being used has a lower clearance: {clientCertDef.SecurityClearance}");
                 }
 
+                if (LoggingSource.AuditLog.IsInfoEnabled)
+                {
+                    var permissions = FormatPermissions(certificate);
+
+                    LogAuditFor("Certificates", "ADD", 
```

Comment:
Under the covers we concatenate action and  target parameters. So this will add a log entry like "ADD Generate certificate ..."

```suggestion
                    LogAuditFor("Certificates", "GENERATE", 
```

Reply:
done